metaserver/src/test/scala/tests/compiler/SquiggliesTest.scala

```diff
@@ -25,7 +34,7 @@ object SquiggliesTest extends CompilerSuite {
       val input = Input.VirtualFile(name, original)
       val doc = Semanticdbs.toSemanticdb(input, compiler)
       val PublishDiagnostics(_, diagnostics) :: Nil =
-        SquiggliesProvider.squigglies(doc, linter)
+        squiggliesProvider.squigglies(doc).runSyncMaybe.right.get
```

Comment:
BTW there is a TestScheduler() which can run in sync mode with `tickOne()`, see how it's used in SymbolIndexTest

Reply:
I remembered you mentioning something like this a few days back, but I got lazy :D
Thanks for the pointer, I'll refactor this into something more appropriate

Reply:
I've kept `runSyncMaybe` but matched over `Right` on the lhs. I'm not sure I understand how the `TestScheduler` would help in this case, tbh.